aio/content/examples/what-is-angular/src/app/hello-world-bindings/hello-world-bindings.component.html

```diff
@@ -4,3 +4,6 @@
 <!-- #docregion bindings -->
 <p [id]="sayHelloId" [style.color]="fontColor">You can set my color in the component!</p>
```

Comment:
I think you can move line number 8 here in the same block no need to create new docregion
```suggestion
<p [id]="sayHelloId" [style.color]="fontColor">You can set my color in the component!</p>
<p>My color is {{ fontColor }}</p>
```

Reply:
FYI it will resolve test case failure as well

Reply:
I think it is best not to add this interpolation to the `bindings` docregion because elsewhere in the guide this snippet is displayed when describing bindings only. Right?

Reply:
yes I see you are correct we are using it in one section just for event binding. So @petebacondarwin I will leave suggestion to you whether you want interpolation word to be removed  or have one more docregion with all 3 examples